README.md

```diff
@@ -97,9 +104,16 @@ Otherwise, the following steps will work for any language:
 
 ## Detailed usage
 
-### Enhancing streaming video with asbplayer features
+First, install the Chrome [extension](https://chromewebstore.google.com/detail/asbplayer-language-learni/hkledmpjpaehamkiehglnbelcpdflcab).
+
+In order to make use of any of asbplayer's features (those outside of subtitles, e.g. mining), subtitles must first be loaded into the extension, with the context of loading them into streaming video or with local video files. The following section instructs how to do this.

```

Comment:
This isn't actually accurate, since asbplayer can be used without installing the extension. This would be more accurate:

> In order to make use of any of asbplayer's features, subtitles (or an empty subtitle track) must first be loaded, either onto a streaming video or a local video file. The following section instructs how to do this.

Even though using more words might be more precise in meaning, in general I think it's clearer to use less, more simple words since it's easier on the reader's eyes. Many people reading this documentation are not English natives so we should try to make it easy for them to read as well.